src/unk_0203A944.c

```diff
@@ -12,1841 +12,1843 @@
 #include "unk_0203A944.h"
 #include "unk_020559DC.h"
 
+#include "constants/overworld_weather.h"
+
 static const u8 Unk_020E98F4[] = {
```

Comment:
Two things:

1. Do these map to individual map headers, as in `include/data/map_headers.h`? If so, can we pre-emptively name this table and also specify indexed initializers on each entry in the array to make the relationship clear?
2. Can we move this to a separate file in `include/data`? e.g., `include/data/map_header_weather.h`

Reply:
1) This isn't injective; there are roughly 3x more elements in this array than map header consts
2) Maybe but that isn't the focus of this PR, I just saw something I could doc quickly